cassandra-release/cassandra-check-release.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Parameters
+# $1 release
+# $2 maven artefacts url (as specified in the vote email)
+#
+# Example use: `./cassandra-check-release.sh 4.0-beta3 https://repository.apache.org/content/repositories/orgapachecassandra-1224/org/apache/cassandra/cassandra-all/4.0-beta3/`
+#
+# This script is very basic and experimental. I beg of you to help improve it.
+#
+
+###################
+# prerequisites
+
+command -v wget >/dev/null 2>&1 || { echo >&2 "wget needs to be installed"; exit 1; }
+command -v gpg >/dev/null 2>&1 || { echo >&2 "gpg needs to be installed"; exit 1; }
+command -v sha1sum >/dev/null 2>&1 || { echo >&2 "sha1sum needs to be installed"; exit 1; }
+command -v md5sum >/dev/null 2>&1 || { echo >&2 "md5sum needs to be installed"; exit 1; }
+command -v sha256sum >/dev/null 2>&1 || { echo >&2 "sha256sum needs to be installed"; exit 1; }
+command -v sha512sum >/dev/null 2>&1 || { echo >&2 "sha512sum needs to be installed"; exit 1; }
+command -v tar >/dev/null 2>&1 || { echo >&2 "tar needs to be installed"; exit 1; }
+command -v timeout >/dev/null 2>&1 || { echo >&2 "timeout needs to be installed"; exit 1; }
+command -v docker >/dev/null 2>&1 || { echo >&2 "docker needs to be installed"; exit 1; }
+(docker info >/dev/null 2>&1) || { echo >&2 "docker needs to running"; exit 1; }
+
+###################
+
+mkdir -p /tmp/$1
+cd /tmp/$1
```

Comment:
An improvement to this script would be to use a docker container instead of the `/tmp` directory.